onnxruntime/core/providers/nuphar/compiler/x86/op_ir_creator/math/quantize/matmul_integer.cc

```diff
@@ -139,7 +140,9 @@ static Status EvaluateMatMulInteger(
         //TODO: change to use MLAS when no layout could apply
         tvm::Tensor B_marshalled = tvm_codegen::Transpose(B, {1, 0});
 
-        tvm::Tensor output_tensor = IMatMulExternMKL(A, B_marshalled, output_shape, input_dim, embed_dim, name + "_IMatMulExternMKL");
+        bool use_extern_MKL = (force_mkl || isAVX);
```

Comment:
>isAVX [](start = 44, length = 5)

should it be !(isAVX2 or isAVX512)? For example, SSE machines may still fail when trying to run AVX2 code.

Reply:
Right, missing those old machines.